platform/linux-generic/odp_comp.c

```diff
@@ -0,0 +1,100 @@
+/* Copyright (c) 2018, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+#include "config.h"
+
+#include <string.h>
+
+#include <odp/api/comp.h>
+#include <odp/api/event.h>
+#include <odp/api/packet.h>
+#include <odp/api/plat/strong_types.h>
+
+#include <odp_debug_internal.h>
+
+void odp_comp_session_param_init(odp_comp_session_param_t *param)
+{
+	memset(param, 0, sizeof(odp_comp_session_param_t));
+}
+
+odp_comp_session_t
+odp_comp_session_create(const odp_comp_session_param_t *params)
+{
+	(void) params;
```

Comment:
here if prior param_init to zero is expected you can add some check to capture missing param_init.

Reply:
We've never done such things before. I see no reason to do it now.